pychord/quality.py

```diff
@@ -76,7 +76,9 @@ def append_on_chord(self, on_chord, root):
             on_chord_val -= 12
 
         if on_chord_val not in components:
-            components.insert(0, on_chord_val)
+            components = [on_chord_val] + [
+                v for v in components if v % 12 != on_chord_val % 12
+            ]
```

Comment:
This is making so there are no repeated components of the same note in different octaves. I'm not sure if this is correct. I added this to make the `test_transpose_slash` test pass.